lib/nexmo_developer/app/views/blog/blogpost/show.html.erb

```diff
@@ -141,7 +138,25 @@
       <% end %>
 
     </div>
-
+        <!-- Call to Action Card -->
+    <div id="card-action" class="Vlt-card Vlt-bg-black">
+      
+      <div class="rectangle"> </div>
+        <span class="square"></span>
+      <div class="title">
+        <h3 class="title-text" >Start Building</h3>
+      </div>
+      <p>Start developing in minutes with free credit on us. No credit card required!</p>
+
+      <a href=<%= blogButtonPath%> class="Vlt-btn-blog-sm">
```

Comment:
Is there a reason why we are using a plain `<a>` tag rather than the `link_to` rails helper?
Something like:
```
<%= link_to blogButtonPath, class: "Vlt-btn-blog-sm" do %> 
     <svg class="Vlt-icon Vlt-icon--smaller Vlt-grey-darker" style="opacity: .6;">
        <use xlink:href="/symbol/volta-icons.svg#Vlt-icon-clock"></use>
      </svg>
<% end %>

Reply:
successfully changed and also changed variable to snake case.